src/main/java/com/example/book/store/config/AppConfig.java

```diff
@@ -0,0 +1,39 @@
+package com.example.book.store.config;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp.BasicDataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@ComponentScan(basePackages = "com.example.book.store")
+public class AppConfig {
```

Comment:
In the first steps, it is better to use the default Beans
You can remove this config